impeller/compiler/shader_lib/impeller/branching.glsl

```diff
@@ -0,0 +1,34 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef BRANCHING_GLSL_
+#define BRANCHING_GLSL_
+
+#include <impeller/constants.glsl>
```

Comment:
Can we settle on a naming for SL code in a header? How about ".sl.h"? Should make it easier for editors and such to highlight by default. And also denote that the file is meant to be included. .glsl is also fine I suppose.